app/services/notification_service.rb

```diff
@@ -23,16 +26,29 @@ def self.invite_to_edit(map, inviter, invited)
     invited.notify(map.invited_text, body, user_map, false, MAILBOXER_CODE_INVITE_TO_EDIT, true, inviter)
   end
 
+  # note: this is a global function, probaobly called from the rails console with some html body
+  def self.message_from_devs(subject, body, opts = {})
+    users = opts[:users] || User.all
+    obj = opts[:obj] || nil
+    sanitize_text = opts[:sanitize_text] || false
+    notification_code = opts[:notification_code] || MAILBOXER_CODE_MESSAGE_FROM_DEVS
+    send_mail = opts[:send_mail] || true
+    sender = opts[:sender] || User.find_by(email: 'ishanshapiro@gmail.com')
+    Mailboxer::Notification.notify_all(users, subject, body, obj, sanitize_text, notification_code, send_mail, sender)
+  end
+
   def self.text_for_notification(notification)
     if notification.notification_code == MAILBOXER_CODE_ACCESS_REQUEST
-      map = notification.notified_object.map
+      map = notification.notified_object&.map
       'wants permission to map with you on <span class="in-bold">' + map.name + '</span>&nbsp;&nbsp;<div class="action">Offer a response</div>'
     elsif notification.notification_code == MAILBOXER_CODE_ACCESS_APPROVED
-      map = notification.notified_object.map
+      map = notification.notified_object&.map
       'granted your request to edit map <span class="in-bold">' + map.name + '</span>'
```

Comment:
This will still throw an error, because of the lines that follow the one that you fixed. The best thing to do might be to filter out notifications in the first place, that don't have a non-nil "notified_object"